### PR TITLE
[Exceedance] calculate exceedances

### PR DESCRIPTION
**Pull request by ja-odur**
#### Summary of Changes (What does this PR do?)
- Calculate exceedances

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Fetch this branch locally.
* Change directory to the `exceedances` micro-service directory.
* Follow the steps in the `README.md` file to run the script.
* The script should run successfully.

#### What are the relevant tickets?
- [N/A]()

### CODE REVIEW

File: src/exceedances/README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@
 
 * Run script
     * `python main.py`
+    * or by explicitly stating the tenant `python main --tenant=kcca`
```
Comment (Baalmart) [inline src/exceedances/README.md:33]: I think we forgot to add the file extension on this entry file....should it be: python main.py --tenant=kcca

**Review by Baalmart — COMMENTED** [review]
Thanks @ja-odur , this looks good. 
I am interesting in witnessing how the deployment will be done, under what environment since default is staging.

Could we also use the same PR to add its corresponding k8s job file under the k8s directory? cc: @danielogen

File: src/exceedances/README.md (hunk 1 above)
Comment (ja-odur) [reply src/exceedances/README.md]: fixed

Comment (Baalmart) [reply src/exceedances/README.md]: Great, thanks!

**Review by Baalmart — COMMENTED** [review]
Thanks @ja-odur 
This LGTM now, just pending the deployment files and we shall be good to go. Unless we create a separate PR just for the deployment files? cc: @danielogen

**Comment by danielogen** [comment]
> Thanks @ja-odur
> This LGTM now, just pending the deployment files and we shall be good to go. Unless we create a separate PR just for the deployment files? cc: @danielogen

Hi @Baalmart , deployment files are coming in a different PR along with some changes to existing k8s files. cc: @ja-odur

**Comment by Baalmart** [comment]
> > Thanks @ja-odur
> > This LGTM now, just pending the deployment files and we shall be good to go. Unless we create a separate PR just for the deployment files? cc: @danielogen
> 
> Hi @Baalmart , deployment files are coming in a different PR along with some changes to existing k8s files. cc: @ja-odur

Alright @danielogen

**Review by Baalmart — APPROVED** [review]
All good...

**Pull request merged**